shopping_mall/backend/app/routers/admin.py

```
@@ -183,11 +212,76 @@ def update_ticket_status(
         )
     t.status = body.status
+
+    # 교환/취소 티켓이 completed로 전환될 때 delivered 주문 → returned 자동 처리
+    if body.status == "completed" and t.action_type in ("cancel", "exchange"):
+        from app.services.order_processor import OrderProcessor
+        order = db.query(Order).filter(Order.id == t.order_id).first()
+        if order and order.status == "delivered":
+            try:
+                OrderProcessor.apply_return(db, order)
+            except ValueError:
+                pass  # 이미 returned이거나 다른 상태 — 조용히 무시
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

(SIM105)